README.md

```diff
@@ -25,32 +25,40 @@ buildscript {
 plugins {
   id "net.ltgt.apt" version "0.11"
   id "net.ltgt.errorprone" version "0.0.11"
-  id "java"
+  id "java-library" // Or just plain old "java"
```

Comment:
the conventions with `java-library` are no longer `compile` etc. This configuration as is will show warnings to the user to switch to `api` and `implementation`. We should be consistent with the example code.

We should either use all the correct conventions of `java-library` or `java` (in this case, `annotationProcessor` is not a valid configuration in the old plugin). So I think it is better to just have two separate examples. One for java and one for android